conn.go

```diff
@@ -1513,6 +1576,31 @@ func (cn *conn) readReadyForQuery() {
 	}
 }
 
+func (c *conn) processBackendKeyData(r *readBuf) {
+	c.processID = r.int32()
+	c.secretKey = r.int32()
+}
+
+func (cn *conn) Cancel() {
+	var err error
+	can := &conn{}
+	can.c, err = dial(cn.dialer, cn.opts)
+	if err != nil {
+		return
```

Comment:
Is there really nothing to be done here? Retry doesn't seem appropriate...

Is this method exported for the driver package? I don't see it in the docs.